consular/main.py

```diff
@@ -10,8 +19,72 @@ class Consular(object):
     def __init__(self, consul_endpoint, marathon_endpoint):
         self.consul_endpoint = consul_endpoint
         self.marathon_endpoint = marathon_endpoint
+        self.pool = HTTPConnectionPool(reactor, persistent=False)
+        self.event_dispatch = {
+            'status_update_event': self.handle_status_update_event,
+        }
+        self.unknown_event_handler = self.handle_unknown_event
```

Comment:
I'm wondering why the rename here? I see `handle_unknown_event` being used below but not `unknown_event_handler`.